mono/mini/debugger-agent.c

```diff
@@ -2734,7 +2727,7 @@ suspend_vm (void)
 
 	if (suspend_count == 1) {
 		// FIXME: Is it safe to call this inside the lock ?
-		start_single_stepping ();
+		mono_de_start_single_stepping ();
```

Comment:
mono_germany_start_single_stepping? 
Can we use dbg or something, rather than "de"?